fe/fe-core/src/main/java/org/apache/doris/ldap/LdapClient.java

```diff
@@ -132,20 +132,20 @@ public static boolean doesUserExist(String userName) {
         return true;
     }
 
-    public static boolean checkPassword(String userName, String password) {
+    boolean checkPassword(String userName, String password) {
         init();
         try {
             clientInfo.getLdapTemplateNoPool().authenticate(org.springframework.ldap.query.LdapQueryBuilder.query()
                     .base(LdapConfig.ldap_user_basedn)
                     .filter(getUserFilter(LdapConfig.ldap_user_filter, userName)), password);
             return true;
-        } catch (Exception e) {
+        } catch (org.springframework.ldap.AuthenticationException e) {
```

Comment:
The range captured here is small, which will result in doris being logged in without a password.